spec.bs

```diff
@@ -32,16 +32,26 @@ A first example illustrates the features of a `tss:Snippet`:
 <snippet/2026-01-01>
   a tss:Snippet;
   tss:points """[
-    { "time": "2026-01-01T06:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>, "value": "5.4"^^<http://www.w3.org/2001/XMLSchema#double>, "id": "https://example.org/0"},
-    { "time": "2026-01-01T06:59:59Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>, "value": "5.2"^^<http://www.w3.org/2001/XMLSchema#double>, "id": "https://example.org/1"},
-    { "time": "2026-01-01T08:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>, "value": "5.2"^^<http://www.w3.org/2001/XMLSchema#double>, "id": "https://example.org/2"},
-    { "time": "2026-01-01T09:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>, "value": "6.1"^^<http://www.w3.org/2001/XMLSchema#double>, "id": "https://example.org/3"},
-  ]"""^^cdt:List;
+    { "time": "2026-01-01T06:00:00Z", "value": "5.4", "id": "https://example.org/0"},
+    { "time": "2026-01-01T06:59:59Z", "value": "5.2", "id": "https://example.org/1"},
+    { "time": "2026-01-01T08:00:00Z", "value": "5.2", "id": "https://example.org/2"},
+    { "time": "2026-01-01T09:00:00Z", "value": "6.1", "id": "https://example.org/3"},
+  ]"""^^rdf:JSON;
   tss:from "2026-01-01T00:00:00Z"^^xsd:dateTime;
   tss:until "2026-01-02T00:00:00Z"^^xsd:dateTime;
   tss:pointType sosa:Observation;
-  tss:timePath sosa:resultTime;
-  tss:valuePath sosa:hasSimpleResult;
+  tss:context """
+      "@context":{
+	   "time": {
+	        "@id":"sosa:resultTime",
```

Comment:
You should then of course also include the prefixes in the context or it won’t work. Better to make this functional!